java-codegen/opensearch-openapi.yaml

```
@@ -44296,6 +44296,8 @@ components:
               type: array
               items:
                 $ref: '#/components/schemas/_common.query_dsl___QueryContainer'
+            pagination_depth:
+              type: number
```

This should match the typing in the spec repo and the code re-generated accordingly
```suggestion
              type: integer
              format: int32
```